website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "React Native for macOS"

```

Comment:
"Announcing React Native for macOS and other updates from React Native @ Microsoft" - the title will get picked up.

Reply:
are you sure? that is... very long:
![image](https://user-images.githubusercontent.com/6964369/82355940-c61bd880-99b7-11ea-9edc-fb205631e9dd.png)

Reply:
how about "announcing React Native for macOS and more" ?

Reply:
changed!